lib/database_cleaner/active_record/deletion.rb

```diff
@@ -66,8 +66,9 @@ def table_stats_query(connection, db_name)
       if @cache_tables && !@table_stats_query.nil?
         return @table_stats_query
       else
+        quote_start, quote_end = connection.quote_table_name('T').split('T')
         @table_stats_query = connection.select_values(<<-SQL).join(' UNION ')
-               SELECT CONCAT('SELECT \"', table_name, '\" AS table_name, COUNT(*) AS exact_row_count FROM ', table_name)
+               SELECT CONCAT('SELECT \"', table_name, '\" AS table_name, COUNT(*) AS exact_row_count FROM #{quote_start}', table_name, '#{quote_end}')
```

Comment:
Yes, take a closer look. `table_name` is not a Ruby variable, inside a Ruby code, that I can use a Ruby function to manipulate. It's an SQL variable used in an SQL function (`CONCAT`) that generates an SQL string. That SQL string needs to have quotes inside it.

I tried `connection.quote_table_name('table_name')` hoping it would affect the SQL string, but it didn't.